.github/workflows/checks.yml

```diff
@@ -0,0 +1,45 @@
+name: Pre-commit Checks
+
+on:
+  push:
+    branches:
+      - 8.*
+      - release-*
+  pull_request:
+
+jobs:
+  bidi_scan_job:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: '32'
+    - name: Run BiDi Scan
+      run: |
+        git fetch origin 8.0
```

Comment:
@altmannmarcelo these should change? and all references to 8.0 in this file?

Reply:
It does, this and clang-format, thankfully we adopted trunk and not 8.1, 8.2, 8.3 ...